### PR TITLE
Remove First in {FirstKey} {FirstStreamReferenceName}

### PR DESCRIPTION
**Pull request by sturnberg**
Changing {FirstKey} to {Key} and {FirstStreamReferenceName} to {StreamReferenceName}.

### CODE REVIEW

File: Content_Portal/Documentation/DataViews/DefineaDataView/Summaries.md [hunk 1]
```diff
@@ -7,15 +7,15 @@ uid: DataViewsSummaries
 
 *Note: This topic covers features that are not yet generally available. If you are interested in trialing these pre-release features, contact your account team for more details.*
 
-Data views can be configured to show calculated summaries for streams and assets.  �Calculated summaries� is defined as the summary types enumerated in [SdsSummaryType](xref:sdsReadingDataApi#get-summaries#get-summaries). This topic describes how to include summaries in a data view. 
+Data views can be configured to show calculated summaries for streams and assets.  �Calculated summaries� is defined as the summary types enumerated in [SdsSummaryType](xref:sdsReadingDataApi#get-summaries#get-summaries). This topic describes how to include summaries in a data view.
```
Comment (osimarkrogers) [inline Content_Portal/Documentation/DataViews/DefineaDataView/Summaries.md:10]: I think "calculated summaries" works better as "summary calculations". Also, either form is plural, so should be "Summary calculations/Calculated summaries are defined...."
One more item, I believe "...summary types enumerated in [SdsSummaryType]..." should be "...the [SdsSummaryType]..." as it is referring to the SDS object.

File: Content_Portal/Documentation/DataViews/DefineaDataView/Summaries.md [hunk 2]
```diff
@@ -7,15 +7,15 @@ uid: DataViewsSummaries
...
 
-In the data view a data field defines the type of summary with the SummaryType property as [SdsSummaryType](xref:sdsReadingDataApi#get-summaries#get-summaries). Another property, SummaryDirection, allows the �summary�s direction� to be defined as [SummaryDirection](xref:ResolvedDataView#SummaryDirection-enumeration). SummaryDirection controls whether the start or end index of the summary is used by the data view to calculate the summary values.
+In the data view a data field defines the type of summary with the SummaryType property as [SdsSummaryType](xref:sdsReadingDataApi#get-summaries#get-summaries). Another property, SummaryDirection, allows the �summary�s direction� to be defined as [SummaryDirection](xref:ResolvedDataView#SummaryDirection-enumeration). SummaryDirection controls whether the start or end index of the summary is used by the data view to calculate the summary values.
```
Comment (osimarkrogers) [inline Content_Portal/Documentation/DataViews/DefineaDataView/Summaries.md:18]: "summary's direction" should be "direction of the summary"

**Review by osimarkrogers — CHANGES_REQUESTED** [review]

File: Content_Portal/Documentation/DataViews/DefineaDataView/Summaries.md (hunk 1 above)
Comment (sturnberg) [reply Content_Portal/Documentation/DataViews/DefineaDataView/Summaries.md]: Made the changes and also fixed a bad link.

File: Content_Portal/Documentation/DataViews/DefineaDataView/Summaries.md (hunk 2 above)
Comment (sturnberg) [reply Content_Portal/Documentation/DataViews/DefineaDataView/Summaries.md]: Made the change and fixed another bad link.

**Review by osimarkrogers — APPROVED** [review]

File: Content_Portal/Documentation/DataViews/DefineaDataView/Field_Sets.md [hunk 3]
```diff
@@ -143,9 +143,9 @@ In cases where the identifiers are unique, the identifier is suffixed with an or
 
 There are four special parameters available for use in field labels:
 - `{IdentifyingValue}` - the value of the identifying field
-- `{FirstKey}` - the value of the first of the `"Keys"` specified on the field
+- `{Key}` - the value of the first of the `"Keys"` specified on the field
```
[the remark this answers is not in the document]
Comment (sturnberg) [reply Content_Portal/Documentation/DataViews/DefineaDataView/Field_Sets.md:146]: Yes, the definition is unchanged. "First" was removed to shorten the parameter but it's still the first key.

[the remark this answers is not in the document]
Comment (osimarkrogers) [reply Content_Portal/Documentation/DataViews/DefineaDataView/Field_Sets.md:146]: All good - - I've already approved as well.

**Pull request merged**